templates/legal/systems-information-notice.html

```diff
@@ -54,9 +54,9 @@ <h2 id="contact">Contact</h2>
       <p>Questions, comments and requests regarding this legal notice are welcomed and should be addressed to <a href="mailto:legal@canonical.com">legal@canonical.com</a> or to the address below:</p>
       <ul class="p-list">
         <li class="p-list__item">Legal, Canonical</li>
-        <li class="p-list__item">5th Floor, Blue Fin Building</li>
-        <li class="p-list__item">110 Southwark Street</li>
-        <li class="p-list__item">London, SE1 0SU</li>
+        <li class="p-list__item">The Office Group, St Dunstans House,</li>
+        <li class="p-list__item">London EC4A 3TW,</li>
```

Comment:
the address is for the contactable address but the post code is for the registered address